src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -98,12 +98,41 @@ public async Task StartContainersAsync(IExecutionContext executionContext, objec
                 await StartContainerAsync(executionContext, container);
             }
 
+            await RunContainersHealthcheck(executionContext, containers);
+        }
+
+        public async Task RunContainersHealthcheck(IExecutionContext executionContext, List<ContainerInfo> containers)
```

Comment:
```suggestion
        private async Task RunContainersHealthcheck(IExecutionContext executionContext, List<ContainerInfo> containers)
```

Reply:
It has to be public to be accessible from the test.